_source/logzio_collections/_shippers/logstash.md

```diff
@@ -122,7 +122,7 @@ Make sure the `mutate` block is the last item in the `filters` block.
 {% include log-shipping/replace-vars.html listener=true %}
```

Comment:
@bgeveritt it won't let me comment on line 67 & 119 - can you change those to `filter` for me please?